app/init.tsx

```diff
@@ -21,27 +18,13 @@ if (process.env.NODE_ENV === 'production') {
   (Sentry as any).captureMessage = (m: any) => console.error(m);
 }
 
-const store = (configureStore as any)();
-const googleAnalyticsId = (process.env.NODE_ENV === 'production' || window.location.host === 'www.askdarcel.org') ? 'UA-116318550-1' : 'UA-116318550-2';
-
-ReactGA.initialize(googleAnalyticsId);
-history.listen(loc => {
-  const page = loc.pathname + loc.search;
-  ReactGA.set({ page });
-  ReactGA.pageview(loc.pathname);
-});
-
 const rootElement = document.getElementById('root')!;
 ReactModal.setAppElement(rootElement);
 
 ReactDOM.render((
-  <Provider store={store} key="provider">
-    <ConnectedRouter history={history}>
-      <ScrollToTop>
```

Comment:
Is `<ScrollToTop>` not necessary anymore?

Reply:
All it did was listen to route changes and perform window.scrollTo(0,0). Felt overkill to be a whole component, I just moved that behavior to a one liner in a useEffect inside <App />

I also added a todo, because I'm not sure we do need it anymore, but that's out of scope for this PR